python/command_runner/dialog.py

```diff
@@ -74,6 +75,12 @@ def _doit(self):
         self.ui.cancel.setText("Close")
         self.ui.cancel.clicked.connect(self.close)
 
+        reload = self._interaction_interface.ask_yn_question(
+            "Would you like to restart " "the engine to reload your changes?"
```

Comment:
Why two strings?

Reply:
hmm good point, could it be that black rearranged things? Or I just messed up 😛